core/src/main/java/com/expediagroup/streamplatform/streamregistry/core/services/ConsumerBindingService.java

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
All entity service classes are updated to use separated `saveSpecification` and `saveStatus` repository methods.